utils/build/docker/java/parametric/run.sh

```diff
@@ -8,9 +8,16 @@ case $(echo "${DD_TRACE_DEBUG:-false}" | tr '[:upper:]' '[:lower:]') in
   *);;
 esac
 
+# TODO Need to disable all instrumentations related to Spring Boot
```

Comment:
I think this line can be removed now ?
Maybe we can convert it into a comment explaining _why_ we disable Spring boot instrumentations. It's obvious in the context of this review, but might not be when looking solely at this file in the future.

Reply:
Yes, that is done but you raised a good point. 
I will document the new flags bellow 👍